docs/sds-plus.md

```diff
@@ -0,0 +1,59 @@
+
+## Key Features of SDS+
+
+### 1. Two-Thirds Release Point for 'Max-Life' Sexual Offences
+
+A key aspect of SDS+ is the introduction of a two-thirds release point for individuals convicted of 'max-life' sexual offences. This release point is triggered when a Standard Determinate Sentence
+(SDS) of 4 years or more is imposed.
+
+### 2. Targeted Release Point Change for Violent Offenders
+After 2022, SDS+ introduced a targeted release point change for violent offenders, particularly those receiving an SDS of 4 to 7 years for 4 Specified violent offences (Manslaughter, Soliciting Murder, Attempt or Conspiracy to Murder).
+
+## Outcome Lists:
+
+1. **List A:**
+   - Schedule 15 (from Criminal Justice Act 2003) Sexual and Violent offences that attracted life between 1 April 2020 and 27 June 2022 inclusive.
+   - https://manage-offences.hmpps.service.justice.gov.uk/schedules/pcsc-lists#list-a
+
+2. **List B:**
+   - Schedule 15 (from Criminal Justice Act 2003) Sexual and 4 specific violent offences identified in the PCSC bill that attracted life on or after 28 June 2022.
+   - https://manage-offences.hmpps.service.justice.gov.uk/schedules/pcsc-lists#list-b
+
+3. **List C:**
+   - Schedule 15 (from Criminal Justice Act 2003) Sexual and 4 specific violent offences identified in the PCSC bill that attracted life on or after 28 June 2022.
+   - https://manage-offences.hmpps.service.justice.gov.uk/schedules/pcsc-lists#list-c
+
+4. **List D:**
+   - Schedule 15 (from Criminal Justice Act 2003) Sexual and Violent offences that attracted life on or after 28 June 2022.
+   - https://manage-offences.hmpps.service.justice.gov.uk/schedules/pcsc-lists#list-d
+
+### SDS+ Identification Conditions:
+
+#### For Standard Determinate (SDS incl ADIMP or ADIMP_ORA) and  YOI (including YOI_ORA) Sentence Types Sentence Types:
+
+- **List A:**
+  - Apply if the sentence type is Standard Determinate (SDS incl ADIMP or ADIMP_ORA).
+  - Identify as SDS+ if the sentence date is before 28/6/22, the sentence length is 7 years or more, and meets the conditions of List A.
+
+- **List B:**
+  - Apply if the sentence type is Standard Determinate (SDS incl ADIMP or ADIMP_ORA).
+  - Identify as SDS+ if the sentence date is on or after 28/6/22, the sentence length is 4 years or more and less than 7 years, and meets the conditions of List B.
+
+- **List D:**
+  - Apply if the sentence type is Standard Determinate (SDS incl ADIMP or ADIMP_ORA).
+  - Identify as SDS+ if the sentence date is on or after 28/6/22, the sentence length is 7 years or more, and meets the conditions of List D.
+
+#### For Section 250 and Section 91 Sentence Types:
+
+- **List A:**
+    - Apply if the sentence type is  SEC250, SEC250_ORA, SEC91_03 and SEC91_03 ORA .
```

Comment:
Slight caveat on this one, Cherky checked with policy over whether SEC91_03 and ORA need to be included in SDS+ and because of when they stopped being issued, they do not.

Please see excerpt from messages with Cherky:
> I've had a response from the Policy team. They can confirm that SEC91_03 and SEC91_03_ORA are not in scope of SDS+ and there is no need to consider age when determining whether SDS+ applies (since the prisoner's age on conviction will determine which sentence type is imposed). I'll update the ACs across the stories to remove SEC91_03 and SEC91_03_ORA sentence types from scope and will add a comment to the stories.